test/parsing/errors/useImportNeedsModule-detailed.good

```diff
@@ -0,0 +1,4 @@
+─── error in useImportNeedsModule.chpl:1 [UseImportNeedsModule] ───
+  'use' statements must refer to module or enum symbols.
+  (e.g., 'use <module>[.<submodule>]*;')
```

Comment:
This can now be a "For example, ", since a whole new sentence in parentheses is kind of awkward.

Reply:
Agree, but now moot as I've removed this example